stories/screen-story.js

```diff
@@ -35,9 +35,12 @@ import Invoice from '../src/view/invoice';
 import InvoiceQR from '../src/view/invoice-qr';
 import Payment from '../src/view/payment';
```

Comment:
Missing PaymentMobile view in storybook?

Reply:
Can't render this one in storybook unfortunately since the react-native-web doesn't have Expo's BarCodeScanner component.